# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/nsGtkKeyUtils.h
# Commit: 6770fb6b8d92
# Full Hash: 6770fb6b8d9217b7d75ed7169cb1389c868498ed
# Author: Martin Stransky <stransky@redhat.com>
# Date: 2024-09-06 21:37:32
# Regressor Bug: 1915596
# File Overlap Count: 2
# Description:
#   Bug 1915596 [Wayland] Get touchpad hold event directly from Wayland display r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D220551
# ==============================================================================

diff -r 791b8e439bbd -r 6770fb6b8d92 widget/gtk/nsGtkKeyUtils.h
--- a/widget/gtk/nsGtkKeyUtils.h	Fri Sep 06 08:37:00 2024 +0000
+++ b/widget/gtk/nsGtkKeyUtils.h	Fri Sep 06 09:16:28 2024 +0000
@@ -198,6 +198,7 @@
    * from xkb_keymap. We call that from Wayland backend routines.
    */
   static void SetModifierMasks(xkb_keymap* aKeymap);
+  static void HandleKeymap(uint32_t format, int fd, uint32_t size);
 
   /**
    * Wayland global focus handlers
@@ -206,10 +207,6 @@
   static void SetFocusOut(wl_surface* aFocusSurface);
   static void GetFocusInfo(wl_surface** aFocusSurface, uint32_t* aFocusSerial);
 
-  static void SetSeat(wl_seat* aSeat, int aId);
-  static void ClearSeat(int aId);
-  static wl_seat* GetSeat();
-
   static void SetKeyboard(wl_keyboard* aKeyboard);
   static wl_keyboard* GetKeyboard();
   static void ClearKeyboard();
@@ -497,9 +494,6 @@
 #endif
 
 #ifdef MOZ_WAYLAND
-  static wl_seat* sSeat;
-  static int sSeatID;
-  static wl_keyboard* sKeyboard;
   wl_surface* mFocusSurface = nullptr;
   uint32_t mFocusSerial = 0;
 #endif